### ISSUE TITLE
Add license check on PRs

### ISSUE THREAD

**Issue by markov00** [elastic/elastic-charts#72]
**Is your feature request related to a problem? Please describe.**
We need to add a license checker that blocks PRs when a incompatible license is found in our dependencies.

**Describe the solution you'd like**
We can use a similar solution to the one used by Kibana, using `license-checker` and a small script to execute it on Travis

**Describe alternatives you've considered**
I didn't find any github app or service that does this check. Only [FOSSA](https://fossa.com) but it has only business plans.

**Comment by wayneseymour** [comment]
@markov00 is this still needed?

**Comment by wayneseymour** [comment]
I didnt realize you all were using TravisCI.  I've used it before though

**Comment by nickofthyme** [comment]
Yeah we used [travis](https://travis-ci.org/github/elastic/elastic-charts) in the past for releasing and publishing but now we use GitHub actions to do that as of about 2 months. We also use [jenkins](https://kibana-ci.elastic.co/job/elastic+elastic-charts+pr-vrts/) to run some PR checks but want to move all that to GH actions at some point.

I believe the intent is to add a GitHub action that is run on all PRs to check license compatibility. In that case, AFAIK this is still needed as there is no GH action to that effect.

Not sure about using the `license-checker` script from kibana or just using [fossa](https://www.npmjs.com/package/fossa) inside a GitHub action. We might need to update `.fossa.yml` to our use case.

**Comment by markov00** [comment]
Thanks Tre/Nick for checking this. Yes, this is still needed. I can send some details on slack, FOSSA is the chosen app for that and there is already a process on how to in clude it with jenkins. We should check if it's possible to use it also in gh action
